Test.cs

```diff
@@ -4,6 +4,8 @@ namespace FactorialCalculator
 {
     public class Factorial
     {
+        private branchName = "Dev"
+        private int id = 10;
         // Method to calculate the factorial of a number
```

Comment:
Consider making the field `id` readonly if it is not meant to be modified after initialization.